.github/workflows/release-next.yml

```diff
@@ -12,12 +38,31 @@ env:
   PROVIDER_URL: ${{ secrets.PROVIDER_URL }}
 
 jobs:
+  # First step: Determine if the commits since last tag require a version bump
+  # Outputs:
+  # - bump_type: The type of semantic version bump (major, minor, patch)
+  # - triggers_bump: 'true' if a version bump is needed, 'false' otherwise
+  check-version-bump:
+    uses: NexusMutual/workflows/.github/workflows/check-version-bump.yml@master
+    with:
+      ref: ${{ github.ref_name }}
+      environment: production
+      bump-command: |
+        timeout 5s npx conventional-recommended-bump --config .github/config/conventional-bump-setup.js
+    secrets:
+      DEPLOYER_APP_ID: ${{ secrets.DEPLOYER_APP_ID }}
+      DEPLOYER_APP_PK: ${{ secrets.DEPLOYER_APP_PK }}
+
+  # Second step: Determine the RC version number
+  # This job only runs if a version bump is needed
+  # It calculates the next RC version based on the new base version and existing RC versions
+  # NOTE: the version bump is not committed to the repository, it is only used to determine the RC version
   rc-version:
-    if: github.event.pull_request.merged == true
+    if: needs.check-version-bump.outputs.triggers_bump == 'true'
```

Comment:
why do we use 'true'

Reply:
with `github.events` the fields can be a boolean

but with job outputs unfortunately it can only be a string - this is a limitation w/in github actions itself